db/seeds.rb

```diff
@@ -1,26 +1,25 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
-
-
-ben = {
-  email: 'benjamin_c_damman@omb.eop.gov',
-  password: 'password',
-  password_confirmation: 'password'
-}
+password = ENV['DEMO_USER_PASSOWRD']
```

Comment:
yes that persists, unless we change the env var. The idea here is just that we are seeding the DB with an account with a password that is not available via git. Since this app will also be behind basic auth, I think that level of security is OK but you know the security standards better than I do!